Game/Pregnancy/MenstrualCycle.gd

```diff
@@ -54,15 +54,24 @@ func getCurrentStage():
 		return CycleStage.Ovulation
 	return CycleStage.LutealPhase
 
+func isEligibleForProlongedPregnancy() -> bool:
+	if(!getCharacter().hasPerk(Perk.FertilityBetterOvulationV3)):
+		return false
+
+	for egg in impregnatedEggCells: #to prevent perpetual pregnancy. If any egg reaches 2.5 progress
+		if(egg.getProgress() >= 2.5): # no further ovulations allowed
```

Comment:
disabling this limit could be another perk, you know :p

Reply:
Madness xd